### DISCUSSION TITLE
Please reconsider the Lazer FPS/MS limit, for the sake of me passing a 7*.

### DISCUSSION THREAD

**Discussion by Caiozix** [ppy/osu#25089] [Ideas] [answered]
Hello , I wanted to know if would be possible to consider removing the lazer's fps limit so that we can have "<1ms", having ">1ms" is giving the pen too much delay.

On stable I got 0.15~0.30ms and that feels amazing, but the 1.0ms on lazer feels SO DELAYED.
I heard explanations of how much of no difference going below 1ms makes to the game, but it does feel different, I do see the cursor delayed, I do feel it delayed, more so when moving slowly or when looking at it.

I understand that there are more concerning matters to give attention and focus to, but this seems like a simple thing and is something that was decided to be limited to 1.000 fps 1ms, so maybe there could be an option that would allow the user to select their fps limit, like stable does... people that have the hardware to reach below 1ms shouldn't be kept from reaching it.

I love osu! but I hate waiting on the cursor to arrive at a circle, the game is already hard enough.
Please remove the fps limit from lazer, I do not want huge fps, I want low ms,  low response time, if you can do that while keeping the fps at 1.000 that fine. Please I need that 0.20ms.

Lazer's ms:
![lazerms](https://github.com/ppy/osu/assets/147595446/88563ca9-1bff-4a0a-a4f1-14e153e93408)

Stable's ms:
![osu ms](https://github.com/ppy/osu/assets/147595446/c716b009-3e4f-440a-8b8e-530fb4ecef14)

Doesn't seem like it change a lot but stable has x4 faster response time, for a constant movement like a pen/tablet, that's a BIG deal, is like playing with 60% smooth when on lazer.

**Comment by peppy** [comment]
Regardless of your intuition, you're noticing some other issue, not the difference between 1ms and whatever you think the game is running at.

The mostly likely cause is going to be the game not entering exclusive fullscreen. Please ensure the game is set to fullscreen and that hitting win-g flashes the screen, rather than opening windows game bar.

If you're only feeling the delay on tablet (please test with mouse) then it may be something specific to your tablet setup, also.

**Reply by Caiozix** [reply, marked as answer]
Thank you for your promptness to respond Peppy.

I honestly wasn't  expecting the issue to be solved on my end, but it was.
There was an update for my tablet/pen, it came out a week ago, and apparently it's solved, it no longer feels like there's a huge smooth on.
I also disabled the "Game bar" thing, it might have helped aswell, I don't know.

Weird thing is that I was only able to notice the delay on Lazer, it felt pretty normal on stable, anyways it's resolved.

Since I got you here Peppy, would it be possible (at the end-game stages of Lazer's development) to add to the Input tab a "import tablet config", for the tablet's area; no matter how hard I try, it is not the same as I'm used to on stable, this makes the transition to Lazer quite dificult to commit because you either get used to the new area and never play stable again or you try to simmulate as close as possible and get frustated when the area doesn't feel the same on stable.

On this pic you can see that the values for X and Y are not 1to1 (the same applies to W and H), this makes it even harder to get right.
![osutablet](https://github.com/ppy/osu/assets/147595446/f323a177-4d67-4598-b75d-9a384c6f4376)

Would also be cool if we had a button or something that the user could click on to check if there are any updates for the tablet they're using. Or maybe just for the tablets that have collab with Osu!, I don't know... it would be cool to click in-game and be sent to their download page to check if there are any new drivers/software for our tablet.

Thank you for helping.

**Reply by peppy** [reply]
Yes we intend to have the ability to input specific numbers in the tablet configuration, at very least.

We already update tablet support ourselves, as we use OpenTabletDriver and they push updates.
